DemoApp/Sources/Components/AppEnvironment.swift

```diff
@@ -57,7 +57,7 @@ extension AppEnvironment {
     static var baseURL: BaseURL = {
         switch configuration {
         case .test:
-            return .demo
+            return .pronto
```

Comment:
Is this required? Video buddy can't use `pronto` anymore, in case we fix it on CI one day, it won't work with this configuration.

Reply:
Ideally we should be able to use any environment. I'm changing it back to demo for now but we can look into it